Applications/Spire/Source/KeyBindings/KeyBindings.cpp

```diff
@@ -45,7 +45,8 @@ void KeyBindings::set(QKeySequence sequence, const Region& region,
       return mapping.m_key_sequence == sequence;
     });
   if(i == m_bindings.end()) {
-    m_bindings.push_back({sequence, Actions(boost::none)});
+    auto binding = KeyBindingMapping{sequence, Actions(boost::none)};
+    m_bindings.push_back(std::move(binding));
     m_bindings.back().m_actions.Set(region, action);
```

Comment:
What I meant here is for you to construct the binding, modify its actions, then push it to the vector.